### PR TITLE
HPCC-11289  Add README file for HPCC Juju Charm Development

### PR DESCRIPTION
**Pull request by xwang2713**
README file include instructions for how to download and work with HPCC Juju Charm application.
The source code is hosted on www.launchpad.net which is the version control environment Ubuntu Juju Charm uses.

### CODE REVIEW

File: charm/README [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Introduction
+------------
+HPCC Juju Charm helps users deploy HPCC Platform on various cloud environment.
+It can be deployed either directly from charm store or on local system image.
+It currently supports deploying HPCC to Ubuntu 12.04 amd64 (precise) and 14.04 amd64 (trusty).
+
+Development system is also on Ubuntu precise or trusty for easily test with local provider.
+
+There are not much differences between precise and trusty codes except HPCC images, checksum
```
Comment (richardkchapman) [inline charm/README]: should be "There are not many differences"

**Comment by xwang2713** [comment]
Correct the phrase in README file and delete HPCC Juju Charm source files

File: charm/README [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+   sudo apt-get update
+   sudo apt-get install charm-tools
+   sudo apt-get bzr
+
+2. Get HPCC Juju Charm source for Ubuntu precise:
+   mkdir precise
+
+   download Download.zip from  https://jujucharms.com/hpcc/precise
+   or
+   charm get cs:precise/spcc
```
Comment (richardkchapman) [inline charm/README]: Is that a typo? Should it be hpcc rather than spcc?

**Comment by xwang2713** [comment]
Corrected it.

**Pull request merged**